trade-finance-manager-ui/server/helpers/string.js

```diff
@@ -15,27 +15,8 @@ const hasValue = (str) => {
 const containsNumber = (str) =>
   /\d/.test(str);
 
-const isAlphanumeric = (str) => {
```

Comment:
Can we leave the function in please, might be useful in future.

Reply:
I'm a bit reluctant to leave this in.

It's quite an odd list of allowed characters that I don't think is likely to be reused.

The regex is also a little strange, it looks like some characters like `/\'` are only allowed at the start of a string but not at the end. And I think it looks like it allows all letters with the `A-Za-z` part, but then also explicitly allows `s` again later on. We'd probably be better off writing some new regex if we did need anything similar in future.

Finally it's not accurately named since it will return `true` for something that isn't alphanumeric like `.,-,.,`, which could cause confusion in future.

Reply:
I'd suggest we remove it if it's not being used. We will forget about it and we could leave junk in the codebase.